Dockerfile

```diff
@@ -0,0 +1,35 @@
+## golang image created 2021-06-24T00:31:06.02014601Z 
+FROM docker.io/library/golang@sha256:be99fa59acd78bb22a41bbc1e15ebfab2262498ee0c2e28c3d09bc44d51d1774 AS builder
+WORKDIR /go/src/github.com/komish/preflight
+COPY . .
+RUN make build
+
+# ubi-minimal image created 2021-06-01T12:12:46.922866Z
+FROM registry.access.redhat.com/ubi8/ubi-minimal@sha256:0ccb9988abbc72d383258d58a7f519a10b637d472f28fbca6eb5fab79ba82a6b
+
+# Add preflight binary
+COPY --from=builder /go/src/github.com/komish/preflight/preflight /usr/local/bin/preflight
+
+# Install dependencies
+RUN microdnf install \
+    buildah \
+    bzip2 \
+    gzip \
+    iptables \
+    openscap-utils \
+    podman \
+    python3-pip \
+    skopeo
+
+# Install OpenShift client binary
+RUN curl -L https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux-4.7.18.tar.gz | tar -xzv -C /usr/local/bin --exclude=README.md --exclude=kubectl
+
+# Install Operator SDK binray
+RUN curl -Lo /usr/local/bin/operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v1.9.0/operator-sdk_linux_amd64
+RUN chmod 755 /usr/local/bin/operator-sdk
+
+# Install operator-courier utility
+RUN pip3 install operator-courier==2.1.11
```

Comment:
I'm not sure we need operator-courier.  Someone can correct me but I think we just need operator-sdk, oscap-podman, oc, podman and maybe opm.

Reply:
This brings up a good point. The operator-courier tests are still part of the test playbooks but I don't think courier works with the new bundle format, so I think that is only for operators using the old bundle format. I am unclear if the current process disables the courier test?

Reply:
I removed the `operator-courier` dependency for now. I will bring it back in as part of the operator-courier verify check if we determine that is needed after all.